authentik/sources/ldap/urls.py

```diff
@@ -1,8 +1,8 @@
 """API URLs"""
 
-from authentik.sources.ldap.api import LDAPPropertyMappingViewSet, LDAPSourceViewSet
+from authentik.sources.ldap.api import LDAPSourcePropertyMappingViewSet, LDAPSourceViewSet
 
 api_urlpatterns = [
-    ("propertymappings/ldap", LDAPPropertyMappingViewSet),
+    ("propertymappings/ldapsource", LDAPSourcePropertyMappingViewSet),
```

Comment:
for providers like entra and gws I started doing `propertymappings/provider/xyz` so I think for this we might as well switch this to `propertymappings/source/ldap`